js/tinymce/plugins/autosave/plugin.js

```
@@ -0,0 +1,234 @@
+/**
+ * TinyMCE version 7.5.2 (TBD)
+ */
+
+(function () {
+    'use strict';
+
+    var global$4 = tinymce.util.Tools.resolve('tinymce.PluginManager');
+
+    const hasProto = (v, constructor, predicate) => {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)